index.d.ts

```diff
@@ -46,12 +46,24 @@ declare module 'react-native-maps' {
     y: number;
   }
 
+  export type EventActionType =
+    | 'marker-press'
+    | 'polygon-press'
+    | 'polyline-press'
+    | 'callout-press'
+    | 'press'
+    | 'long-press'
+    | 'overlay-press'
+    | undefined;
```

Comment:
Is there ever a case where the action will be undefined ?

Reply:
Yep @christopherdro , actually the value results `undefined` when pressing outside of any specific element (marker, polygon or any other)